documentation/onboard-service.md

```diff
@@ -1,4 +1,18 @@
-# Onboard a new service to AKS
+# Onboard a new service to the Teacher Services AKS cloud platform
+
+Onboarding is pretty straightforward and quick..
+```mermaid
+flowchart TD;
+    Start["Start"] --> Slack["Say Hi in DfE Slack channel #teacher-services-infra"];
+    Slack --> GetCoding["Start generating the code using our template"];
+    Slack --> FillForm["Read this page.. /documentation/onboard-form-template.md <br> and make sure you have all the info"];
+    FillForm --> OnboardCall["Arrange a quick intro call via DfE Slack channel #teacher-services-infra <br> to get things explained and meet the team"];
+    OnboardCall --> SetUpCall["We will help you request access, and set you up to self serve"];
+    SetUpCall --> SetUpComplete["Secure access to platform complete"];
+    GetCoding --> SetUpComplete;
+    SetUpComplete --> Done["Start deploying"] ;
+```
```

Comment:
Can we add a final step to check the [production-checklist](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/production-checklist.md)

Reply:
sure thing